react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class AccordionSegment extends React.Component {
+  render() {
+    return (this.props.content === '')
+      ? (
+        <div id={this.props.identifier} className="accordion-segment">
+          <h2 onClick={this.props.handleClick} className="segment-title">{this.props.title}</h2>
+        </div>
+        )
+      : (
+        <div id={this.props.identifier} className="accordion-segment">
+          <h2 onClick={this.props.handleClick} className="segment-title">{this.props.title}</h2>
+          <p className="segment-content">{this.props.content}</p>
+        </div>
+        );
+  }
+}
```

Comment:
This should just be a function component, also you can simplify this a bit to reduce duplication:

```JSX
function AccordionSegment(props) {
  return (
    <div id={props.identifier} className="accordion-segment">
      <h2 onClick={props.handleClick} className="segment-title">{props.title}</h2>
      {
        props.content === ''
          ? null
          : <p className="segment-content">{props.content}</p>
      }
    </div>
  );
}